lib/cli.js

```diff
@@ -130,13 +130,6 @@ const meowOptions = {
 
         Pattern of files to ignore (in addition to those in .stylelintignore)
 
-      --syntax, -s
-
-        Specify a syntax. Options: "css", "css-in-js", "html", "less",
-        "markdown", "sass", "scss", "sugarss". If you do not specify a syntax,
-        syntaxes will be automatically inferred by the file extensions
-        and file content.
-
       --fix
 
         Automatically fix violations of certain rules.
```

Comment:
[suggestion] We can remove the followings:

https://github.com/stylelint/stylelint/blob/5dfe07c1ba44425227b2c25992663916ff8566f1/lib/cli.js#L44

https://github.com/stylelint/stylelint/blob/5dfe07c1ba44425227b2c25992663916ff8566f1/lib/cli.js#L79

https://github.com/stylelint/stylelint/blob/5dfe07c1ba44425227b2c25992663916ff8566f1/lib/cli.js#L312-L315

Reply:
Aren't these still needed for showing a warning message when `--syntax` is used?

Reply:
Ah, surely 👍🏼